### PR TITLE
Chat updates

### PR DESCRIPTION
**Pull request by gyermolenko**
back:
- test index view (send/receive ws message)
- generate random username with `faker` (strings like Tadfafpqouqpwer are hard to read)
- substitute `send_str` with `send_json`, remove explicit loop

front:
- disable send btn when disconnected
- prepend timestamp as prompt before chat message

### CODE REVIEW

File: demos/chat/aiohttpdemo_chat/views.py [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 import aiohttp
+import aiohttp_jinja2
 from aiohttp import web
-
+from faker import Faker
 
 log = logging.getLogger(__name__)
 
 
+def random_name():
```
Comment (webknjaz) [inline demos/chat/aiohttpdemo_chat/views.py]: `get_random_name()`

File: demos/chat/tests/test_chat.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    assert ack_msg1.json()['action'] == 'connect'
+    ack_msg2 = await ws2.receive()
+    assert ack_msg2.json()['action'] == 'connect'
+
+    text_to_send = 'hi all'
+    await ws1.send_str(text_to_send)
+    received_msg = await ws2.receive()
+    received_dict = received_msg.json()
+
+    assert set(received_dict) == {'action', 'name', 'text'}
```
[the remark this answers is not in the document]
Comment (gyermolenko) [reply demos/chat/tests/test_chat.py]: Here we know only keys. `action` is an implementation detail, `name` is random, and `text` gets compared in the next line

File: demos/chat/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import pytest
+from aiohttpdemo_chat.main import init_app
+
+
+@pytest.fixture
+async def cli(test_client):
```
[the remark this answers is not in the document]
Comment (gyermolenko) [reply demos/chat/tests/conftest.py]: agree

File: demos/chat/aiohttpdemo_chat/views.py (hunk 1 above)
Comment (gyermolenko) [reply demos/chat/aiohttpdemo_chat/views.py]: agree

File: demos/chat/tests/test_chat.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (webknjaz) [reply demos/chat/tests/test_chat.py]: AFAICS `received_dict['name'] == ack_msg1['name']` and `received_dict['action'] == 'sent'`

**Pull request merged**